backend/audit/etl.py

```diff
@@ -144,52 +162,50 @@ def load_federal_award(self):
-                planned_action=entry["planned_action"],
-            )
-            cap_text.save()
+            in corrective_action_plan["CorrectiveActionPlan"]
+        ):
+            corrective_action_plan_entries = corrective_action_plan[
+                "CorrectiveActionPlan"
+            ]["corrective_action_plan_entries"]
+            for entry in corrective_action_plan_entries:
+                cap_text = CapText(
+                    report_id=self.report_id,
+                    finding_ref_number=entry["reference_number"],
+                    contains_chart_or_table=entry["contains_chart_or_table"] == "Y",
+                    planned_action=entry["planned_action"],
+                )
+                cap_text.save()
 
     def load_note(self):
         notes_to_sefa = self.single_audit_checklist.notes_to_sefa["NotesToSefa"]
-        if not notes_to_sefa:
-            logger.warning("No notes to sefa found to load")
-            return
-
-        accounting_policies = notes_to_sefa["accounting_policies"]
-        is_minimis_rate_used = notes_to_sefa["is_minimis_rate_used"] == "Y"
-        rate_explained = notes_to_sefa["rate_explained"]
-        entries = notes_to_sefa["notes_to_sefa_entries"]
-        if not entries:
-            note = Note(
-                report_id=self.report_id,
-                accounting_policies=accounting_policies,
-                is_minimis_rate_used=is_minimis_rate_used,
-                rate_explained=rate_explained,
-            )
-            note.save()
-        else:
-            for entry in entries:
-                note = Note(
-                    report_id=self.report_id,
-                    note_seq_number=entry["seq_number"],
-                    content=entry["note_content"],
-                    note_title=entry["note_title"],
-                    accounting_policies=accounting_policies,
-                    is_minimis_rate_used=is_minimis_rate_used,
-                    rate_explained=rate_explained,
-                )
-                note.save()
+        if notes_to_sefa:
+            accounting_policies = notes_to_sefa["accounting_policies"]
+            is_minimis_rate_used = notes_to_sefa["is_minimis_rate_used"] == "Y"
+            rate_explained = notes_to_sefa["rate_explained"]
+            if "notes_to_sefa_entries" in notes_to_sefa:
+                entries = notes_to_sefa["notes_to_sefa_entries"]
+                if not entries:
+                    note = Note(
+                        report_id=self.report_id,
+                        accounting_policies=accounting_policies,
+                        is_minimis_rate_used=is_minimis_rate_used,
+                        rate_explained=rate_explained,
+                    )
+                    note.save()
+                else:
+                    for entry in entries:
+                        note = Note(
+                            report_id=self.report_id,
+                            note_seq_number=entry["seq_number"],
```

Comment:
Where does the `seq_number` come from in this code? We don't have it in the workbooks.